<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutomaticAnnotationWidget</class>
 <widget class="QMainWindow" name="AutomaticAnnotationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayoutCentral">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitterTB">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayoutTop">
        <item row="0" column="0" colspan="2">
         <widget class="QSplitter" name="splitterLR">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>6</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QDockWidget" name="dockCorpusExplorer">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>6</verstretch>
            </sizepolicy>
           </property>
           <property name="features">
            <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
           </property>
           <property name="windowTitle">
            <string>Corpus items to annotate</string>
           </property>
           <widget class="QWidget" name="dockWidgetCorpusExplorer">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>8</verstretch>
             </sizepolicy>
            </property>
            <layout class="QGridLayout" name="gridLayoutCorpusExplorer">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </widget>
          <widget class="QDockWidget" name="dockAnnotationPlugins">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>6</verstretch>
            </sizepolicy>
           </property>
           <property name="windowTitle">
            <string>Annotation Plugins</string>
           </property>
           <widget class="QWidget" name="dockWidgetContentsAnnotationPlugins">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>8</verstretch>
             </sizepolicy>
            </property>
            <layout class="QGridLayout" name="gridLayoutAnnotationPlugins">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </widget>
          <widget class="QDockWidget" name="dockPluginParameters">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>6</verstretch>
            </sizepolicy>
           </property>
           <property name="windowTitle">
            <string>Plugin Parameters</string>
           </property>
           <widget class="QWidget" name="dockWidgetPluginParameters">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>8</verstretch>
             </sizepolicy>
            </property>
            <layout class="QGridLayout" name="gridLayoutPluginParameters">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCommandLinkButton" name="commandAnnotate">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Annotate!</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QProgressBar" name="progressBar">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QDockWidget" name="dockWidgetOutput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="windowTitle">
        <string>Annotation Plugins Output</string>
       </property>
       <widget class="QWidget" name="dockWidgetContentsOutput">
        <layout class="QGridLayout" name="gridLayoutOutput">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
